packages/flame/lib/src/game/flame_component_system_game.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:flame/src/components/core/component.dart';
+import 'package:flame/src/components/core/component_tree_root.dart';
+import 'package:meta/meta.dart';
+
+/// An interface which should be implemented by any `Game` subclass that uses
+/// the Flame Component System (i.e. it contains a single Component tree).
+@internal
+abstract class FlameComponentSystemGame {
```

Comment:
Pretty much everything in `FlameGame` is FCS + `Game`, do we really need this then?
If someone wants to implement their own `FCS` they can just do `implements FlameGame`?

Reply:
We'd need to have this for the `BareFlameGame` class, so that it can share the interface with `FlameGame`, but without inheriting any unnecessary stuff.

Reply:
I don't think we should consider `BareFlameGame` before it exists, it may very well never come into existence and we deprecate the camera from `FlameGame` before. And if it does come into existence it is better to add this interface on that PR.

Reply:
Ok, I removed this class for now.